ui/public/locales/en.json

```diff
@@ -3285,6 +3285,7 @@
 "message.set.default.nic": "Please confirm that you would like to make this NIC the default for this Instance.",
 "message.set.default.nic.manual": "Please manually update the default NIC on the Instance now.",
 "message.setting.updated": "Setting Updated:",
+"message.setting.update.delay": "The new value will take effect within 30 seconds.",
```

Comment:
@shwstppr Since we are invalidating the cache, do we need this message?

Reply:
@vishesh92 on a multi-management server env other MS will still take 30s to reflect the change